runtests.py

```diff
@@ -473,7 +473,10 @@ def get_openmp_compiler_flags(language):
                                            'run.unicode_imports',  # encoding problems on appveyor in Py2
                                            'run.reimport_failure',  # reimports don't do anything in Py2
                                            'run.cpp_stl_cmath_cpp17',
-                                           'run.cpp_stl_cmath_cpp20'
+                                           'run.cpp_stl_cmath_cpp20',
+                                           'run.large_integer_T5290',  # Py2 adds an L at the end of
```

Comment:
I've taken the view that it isn't a huge problem if a few variables have different names when run with Python 2 so just disabled the test

Reply:
This could probably stay enabled if we used the same "strip suffix in Py2" helper function to build the names.

Reply:
Yes - I'm pretty sure this should work now actually so I'll re-enable it.